src/pages/questionPage.js

```diff
@@ -20,13 +23,17 @@ export const initQuestionPage = () => {
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
   answersListElement.classList.add('answers-list');
 
+  // Append realtime score panel into DOM
+  const scorePanel = createScorePanel(quizData.score, quizData.questions.length);
```

Comment:
There seems to be some overlap with #16 buy @xrustik666. Please communicate with each other to resolve this.